opentech/static_src/src/sass/components/_form.scss

```diff
@@ -4,7 +4,7 @@
         border-bottom: 1px solid $color--white;
     }
 
-    &--login {
+    &--with-p-tags {
```

Comment:
could this be made general. i.e. `form p { ... }` ?

Reply:
Yes, potentially. We'd need to check what affect this would have where other forms are used - search, funds, footer signup etc